### ISSUE TITLE
idea's for a name

### ISSUE THREAD

**Issue by jhjacobs81** [linuxkettle/AppShop#5]
A simple google query shows lots of appshop names already, we need something simple but reckognizable.

**Comment by jhjacobs81** [comment]
something alone the line of Giraphe ?

like.. they are tall, they oversee everything.. like our store oversees all the application messes? 

"Giraphe, the last appstore you'll ever need!"

**Comment by kettle-7** [comment]
I'm worried that will come up with pictures of giraffes.

[Check this out](https://github.com/linuxkettle/LinuxKettle/tree/main/RandomWords_GTK#randomwords_gtk--randomwords_qt). It is adequate to annoy an English teacher.

**Comment by jhjacobs81** [comment]
we could make the giraffe our mascotte ;-) It should be a simple, easy to remember name. But at the same time could have a single, very pretty logo. (yes, like Apple)

**Comment by kettle-7** [comment]
Anyway, "App Shop" is too generic of a name.

**Comment by jhjacobs81** [comment]
it is! But its a nice working title ;-) Elementary has Pantheon already, otherwise that would be a good name :) we'll brainstorm a bit about this :) 

isnt it very late overthere? (coming to think of it.. im not even sure where "there" is)

I'll set up a matrix server on my own domain (swimmingturtle.eu) later this afternoon :) We can use it untill such time we found a good name that we can use for the long term :)

**Comment by kettle-7** [comment]
> isnt it very late overthere?

No, New Zealand time is UTC+12 in winter, and +13 in summer. It's winter now.

> I'll set up a matrix server on my own domain (swimmingturtle.eu) later this afternoon :) We can use it untill such time we found a good name that we can use for the long term :)

Your own domain - fancy! I should be on there tomorrow.

> Elementary has Pantheon already

They don't have Parthenon though!  
(ignore above sentence)

**Comment by kettle-7** [comment]
swimmingturtle.eu redirects me to a login page on sospora.com

**Comment by jhjacobs81** [comment]
yeah i know! i tried setting up a diaspora server once :D wasn't a big succes..

need to fix that still.

SwimmingTurtle was supposed to be the software branch of my IT company :)

**Comment by kettle-7** [comment]
I just use Github pages at the moment, there's not much on there at the moment, I had a blog about gtk programming on linux, but now I do qt programming on freebsd!

I didn't know much about FreeBSD until about a month ago, I knew it was out there but not much more. I've already gotten used to Alt shortcuts to the extent that Fedora is now almost unusable!

**Comment by jhjacobs81** [comment]
in due time.. FreeBSD can be the base for our own macOS Fork :P and you can be the project leader :D

**Comment by kettle-7** [comment]
Sounds delicious!

**Comment by kettle-7** [comment]
> our store oversees all the application messes?

Giraffe. Sees. Everything.

**Comment by jhjacobs81** [comment]
;-)

there’s a matrix server on matrix.swimmingturtle.eu ;-) no web client though

**Comment by ghost** [comment]
Hi, maybe a name like AppShelf.

**Comment by jhjacobs81** [comment]
I like that :)

**Comment by ghost** [comment]
Or FrontShelf

**Comment by kettle-7** [comment]
Nice :)

**Comment by jhjacobs81** [comment]
I think Appshelf is better, it makes it clear what it does..

frontshelf.. front to what?? New email? Settings??

**Comment by ghost** [comment]
I get it. I also like AppShelf better.

**Comment by kettle-7** [comment]
'Tis thy shelf of apps.

**Comment by ghost** [comment]
Maybe close this one, no?
